### PR TITLE
Make JDBC Driver configurable

### PR DESCRIPTION
**Pull request by maxbraun**
Hey,

we've different DBMS preconfigured, which we'd like to leverage. This chart is used to work with postgres. With minor adjustments it could be open to different DMBS's as well.

Hope this is not too quick & dirty.

Appreciate any feedback.

Cheers, Max

### CODE REVIEW

File: charts/sonarqube/templates/postgres-config.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ data:
   SONARQUBE_JDBC_USERNAME:  {{ .Values.postgresql.postgresqlUsername | quote }}
 {{- end }}
 {{- if and .Values.postgresql.service.port .Values.postgresql.postgresqlDatabase }}
-  SONARQUBE_JDBC_URL: "jdbc:postgresql://{{ template "postgresql.hostname" . }}:{{- .Values.postgresql.service.port -}}/{{- .Values.postgresql.postgresqlDatabase -}}"
+  SONARQUBE_JDBC_URL: "jdbc:"{- .Values.JdbcDatabaseType -}"://{{ template "postgresql.hostname" . }}:{{- .Values.postgresql.service.port -}}/{{- .Values.postgresql.postgresqlDatabase -}}"
```
Comment (rjkernick) [inline charts/sonarqube/templates/postgres-config.yaml:15]: Please remove quotes from added parameter

File: charts/sonarqube/values.yaml [hunk 2]
```diff
@@ -196,6 +196,9 @@ plugins:
 # The 'sonar.secretKeyPath' property will be set automatically.
 # sonarSecretKey: "settings-encryption-secret"
 
+## JDBC Database Type; by default postgresql. To use a different Database type, adjust
+JdbcDatabaseType: postgresql
```
Comment (rjkernick) [inline charts/sonarqube/values.yaml:200]: Please change variable to use camelcase. ie: jdbcDatabaseType

File: charts/sonarqube/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: SonarQube is an open sourced code quality scanning tool
-version: 6.4.2
+version: 6.4.3
```
Comment (rjkernick) [inline charts/sonarqube/Chart.yaml:4]: Please bump the minor version instead as this adds a feature

**Comment by maxbraun** [comment]
Thanks @rjkernick. Adjusted according to your comments.

**Pull request merged**